drupal/sites/default/boinc/modules/boincwork/includes/boincwork.helpers.inc

```diff
@@ -203,6 +203,17 @@ function boincwork_get_project_specific_config() {
   
   // Convert XML to array for validation
   $xml = load_configuration($raw_config_data);
+  // By default the returned array should have an 'app_overrides'
+  // element. Even if the admin did not specificy an <apps> container
```

Comment:
Did you mean `Even if the admin did not specificy an <app_overrides> container`?